tests/contract/test_registry_management.py

```diff
@@ -120,7 +118,7 @@ def test_normal_deregister(smc_handler):  # noqa: F811
     assert notary_pool_length == 1
 
     # Fast foward
-    mine(w3, 100 - w3.eth.blockNumber)
+    mine(w3, 20 - w3.eth.blockNumber)
```

Comment:
This should probably be replaced by `fast_forward`. My bad. Forgot to update this part.